joomla/joomla-cms
# [4.0] Replace com_content 'Image Float' with 'Image Class

This is a rebase of #17402. Opened on request https://github.com/joomla/joomla-cms/issues/30924#issuecomment-705354364

Pull Request for Issue #17399 #30924 .

### Summary of Changes
Replaces the 'Image Float ' field in com_contents with an 'Image Class' field

### Testing Instructions
Navigate to the **Images and Links** tab in article edit. Set an image and add a utility class to the **Image Class** field . Check frontend and ensure class has been applied correctly.
